artifactory/utils/buildinfoproperties.go

```diff
@@ -68,6 +68,10 @@ const USE_GRADLE_PLUGIN = "usePlugin"
 const USE_GRADLE_WRAPPER = "useWrapper"
 const FORK_COUNT = "forkCount"
 
+const INCLUDE_PATTERNS = "includePatterns"
+const EXCLUDE_PATTERNS = "excludePatterns"
+const FILTER_EXCLUDED_ARTIFACTS_FROM_BUILD = "filterExcludedArtifactsFromBuild"
```

Comment:
Let's change the name format of all constants in this file to match go's convention. For examine - 
FILTER_EXCLUDED_ARTIFACTS_FROM_BUILD --> filterExcludedArtifactsFromBuild